### PR TITLE
Create 2021-05-10-Diver-com-paper.md

### PR DESCRIPTION
**Pull request by Florie25**

### CODE REVIEW

**Comment by Florie25** [comment]
Hi @AnneThKu , could you check this PR before the one for the UnetStack release, as @esthertanhj needs it for social media for this week.
Thank you
Florie

File: _posts/2021-05-10-Diver-com-paper.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+banner : images/banner-pulse-standard.jpg
+date:  2021-05-10
+categories: wnc; news
+thumbnail: images/pulse-thumbnail-Diver-Comm-05-2021.jpg
+---
+<div class='pulse-img-div'>
+    <img src="{{site.baseurl}}/images/pulse-thumbnail-Diver-Comm-05-2021.jpg" class='pulse-img'>
+</div>
+
+We are very proud to announce that the our paper about [Diver communication and localization paper](https://ieeexplore.ieee.org/abstract/document/9389462) has been published by IEEE Xplore.
```
Comment (manuignatius) [inline _posts/2021-05-10-Diver-com-paper.md]:
```suggestion
We are proud to share that the our paper titled [_Diver Communication and Localization System using Underwater Acoustics_](https://ieeexplore.ieee.org/abstract/document/9389462) has been published in IEEE Xplore.
```

File: _posts/2021-05-10-Diver-com-paper.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+The paper is the fruit of several months of work across the Subnero team and the collaboration of [Professor Mandar Chitre](https://arl.nus.edu.sg/people/mandar-chitre/){:target="_blank"}, of the [Acoustic Research Laboratories (ARL)](https://arl.nus.edu.sg/){:target="_blank"},
```
Comment (manuignatius) [inline _posts/2021-05-10-Diver-com-paper.md]:
```suggestion
The paper is the fruit of several months of work by the Subnero team, in collaboration with [Dr. Mandar Chitre](https://arl.nus.edu.sg/people/mandar-chitre/){:target="_blank"}, of the [Acoustic Research Laboratories (ARL)](https://arl.nus.edu.sg/){:target="_blank"},
```

File: _posts/2021-05-10-Diver-com-paper.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+As the title indicates, the team has developed on an underwater communication system to improve divers safety using our off the shelf software-defined as well as newly designed devices.
```
Comment (manuignatius) [inline _posts/2021-05-10-Diver-com-paper.md]:
```suggestion
As the title indicates, the team used our software-defined underwater acoustic communication and localization technology to develop a diver communication and localization system to improve operational safety of underwater divers.
```

File: _posts/2021-05-10-Diver-com-paper.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Diver communication and localization paper published
```
Comment (manuignatius) [inline _posts/2021-05-10-Diver-com-paper.md]:
```suggestion
title: Diver Communication and Localization System using Underwater Acoustics
```

File: _posts/2021-05-10-Diver-com-paper.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+banner : images/banner-pulse-standard.jpg
+date:  2021-05-10
+categories: wnc; news
+thumbnail: images/pulse-thumbnail-Diver-Comm-05-2021.jpg
```
Comment (manuignatius) [inline _posts/2021-05-10-Diver-com-paper.md]: Please correct the filename to have all lower case letters

File: _posts/2021-05-10-Diver-com-paper.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    <img src="{{site.baseurl}}/images/pulse-thumbnail-Diver-Comm-05-2021.jpg" class='pulse-img'>
+</div>
+
+We are very proud to announce that the our paper about [Diver communication and localization paper](https://ieeexplore.ieee.org/abstract/document/9389462) has been published by IEEE Xplore.
+
+The paper is the fruit of several months of work across the Subnero team and the collaboration of [Professor Mandar Chitre](https://arl.nus.edu.sg/people/mandar-chitre/){:target="_blank"}, of the [Acoustic Research Laboratories (ARL)](https://arl.nus.edu.sg/){:target="_blank"},
+
+As the title indicates, the team has developed on an underwater communication system to improve divers safety using our off the shelf software-defined as well as newly designed devices.
+
+The paper was presented virtually by Dr Prasad Anjangi during [Oceans 2020](https://global20.oceansconference.org/) back in October 2020.
```
Comment (manuignatius) [inline _posts/2021-05-10-Diver-com-paper.md]:
```suggestion
The paper was presented virtually by Dr. Prasad Anjangi during [Global Oceans 2020](https://global20.oceansconference.org/) back in October 2020.
```

**Review by manuignatius — CHANGES_REQUESTED** [review]

File: _posts/2021-05-10-Diver-com-paper.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+banner : images/banner-pulse-standard.jpg
+date:  2021-05-10
+categories: wnc; news
+thumbnail: images/pulse-thumbnail-Diver-Comm-05-2021.jpg
+---
+<div class='pulse-img-div'>
+    <img src="{{site.baseurl}}/images/pulse-thumbnail-Diver-Comm-05-2021.jpg" class='pulse-img'>
+</div>
+
+We are proud to share that our paper titled [_Diver Communication and Localization System using Underwater Acoustics_](https://ieeexplore.ieee.org/abstract/document/9389462) has been published in IEEE Xplore.
```
Comment (AnneThKu) [inline _posts/2021-05-10-Diver-com-paper.md]:
```suggestion
We are proud to share that our paper titled [_Diver Communication and Localization System using Underwater Acoustics_](https://ieeexplore.ieee.org/abstract/document/9389462){:target="_blank"} has been published in IEEE Xplore.
```

File: _posts/2021-05-10-Diver-com-paper.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+The paper is the fruit of several months of work by the Subnero team, in collaboration with [Dr. Mandar Chitre](https://arl.nus.edu.sg/people/mandar-chitre/){:target="_blank"}, of the [Acoustic Research Laboratories (ARL)](https://arl.nus.edu.sg/){:target="_blank"},
+
+As the title indicates, the team used our software-defined underwater acoustic communication and localization technology to develop a diver communication and localization system to improve operational safety of underwater divers.
+
+The paper was presented virtually by Dr. Prasad Anjangi during [Global Oceans 2020](https://global20.oceansconference.org/) back in October 2020.
```
Comment (AnneThKu) [inline _posts/2021-05-10-Diver-com-paper.md]:
```suggestion
The paper was presented virtually by [Dr. Prasad Anjangi](https://subnero.com/people/prasad/) during [Global Oceans 2020](https://global20.oceansconference.org/) back in October 2020.
```

Comment (AnneThKu) [inline _posts/2021-05-10-Diver-com-paper.md]: Do we have the link for the presentation? If so, maybe that would be more relevant than the given oceans link.

File: _posts/2021-05-10-Diver-com-paper.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+For more details, dowlonad the paper in the link above or [contact us](info@subnero.com).
```
Comment (AnneThKu) [inline _posts/2021-05-10-Diver-com-paper.md]:
```suggestion
Download the full paper in the link above or [contact us](https://subnero.com/contact/) for more details.
```

**Review by AnneThKu — CHANGES_REQUESTED** [review]

File: _posts/2021-05-10-Diver-com-paper.md (hunk 8 above)
Comment (Florie25) [reply _posts/2021-05-10-Diver-com-paper.md]: The link, at the moment, is on our Gdrive. Is there a way to share it without giving access to our Drive?

**Review by AnneThKu — APPROVED** [review]

**Review by manuignatius — APPROVED** [review]

**Pull request merged**